Makefile

```diff
@@ -80,14 +80,14 @@ install-develop: clean-build clean-pyc ## install the package in editable mode a
 # LINT TARGETS
 
 .PHONY: lint
-lint: ## check style with flake8 and isort
-	invoke lint
```

Comment:
since you updated the lint command in tasks.py you can just leave this

Reply:
Fixed:
https://github.com/sdv-dev/RDT/pull/809/commits/2c4a754ab473d822d529cd81624d07a3520a6bd9